Socket.io

ROEL2407 opened the pull request:
socket.io toegevoegd en werkend gekregen.

ROEL2407 committed:
socket.io getting started

ROEL2407 committed:
socket.io toegevoegd

ROEL2407 committed:
Merge branch 'main' of https://github.com/M4TThys123/Blok-Tech-Team into Socket.io

timmit147 reviewed (changes requested):
Code ziet er goed uit index.html inline css alleen nog in style.css zetten.

ROEL2407 commented:
> Code ziet er goed uit index.html inline css alleen nog in style.css zetten.

Moest er sowieso uit bedenk ik me net, dacht dat ik dat al gedaan had

ROEL2407 committed:
Delete index.html

Was als test om socket aan de praat te krijgen maar uit eindelijk was het niet nodig

marloestacx reviewed (approved):
Ziet er goed uit

timmit147 reviewed (approved):
Ziet er goed uit.
